File: C/EX17.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  int A, B, C;
  
  printf("\n Digite o primeiro valor: ");
  scanf("%i", &A);
 
  printf("Digite o segundo velor: ");
  scanf("%i", &B);
 
  printf("Digite o terceiro valor: ");
  scanf("%i", &C);
 
  if ( A > B )
  {
           if ( A > C )
            {
               if (A*A==((B*B)+(C*C)))
               {
                   printf("Triangulo retangulo!");
               }
               else
                {
                printf("Nao e um triangulo retangulo");  
                }
           }
           else if (C*C==((A*A)+(B*B)))
           {
            printf("Triangulo retangulo!");
           }
            else
            {
             printf("Nao e um triangulo retangulo");     
            }
 }
else if (B > C)
    {
        if (B*B==((A*A)+(C*C)))
        {
         printf("Triangulo retangulo!");  
        }
        else
        {
        printf("Nao e um triangulo retangulo"); 
        }
    }
else if (C*C==((A*A)+(B*B)))
    {
    printf("Triangulo retangulo!");
    } 
    else 
    {
   printf ("Nao e um triangulo retangulo");  
    }
  return 0;
}

File: Python/Ex17.py
a = float(input("Digite o valor do primeiro lado: "))
b = float(input("Digite o valor do segundo lado: "))
c = float(input("Digite o valor do terceiro lado: "))

if (a > b):
        if (a > c):
            if (a*a == (b*b)+(c*c)):
                print ("Triângulo")
            else:
                print ("Não é um triângulo")
        elif (c*c==(a*a)+(b*b)):
            print ("Triângulo")
        else:
            print ("Não é um triângulo")   
elif b > c:
        if (b*b == (a*a)+(c*c)):
            print ("Triângulo")
        else:
            print ("Não é um triângulo")
elif (c*c==(a*a)+(b*b)): 
    print ("Triângulo")
else:
    print ("Não é um triângulo")
